daily_milestone.go

```diff
@@ -85,7 +86,53 @@ func getProjectID(baseURL string, Token string, Projectname string, Namespace st
 	return strconv.Itoa(project.ID)
 }
 
-// CreateMilestoneData is used to check the due date using the time package of python
+func getMilestones(baseURL string, token string, projectID string) []string {
+	project := gitLabAPI{}
+	list := []string{}
+	strurl := []string{"https://", baseURL, "/projects/", projectID, "/milestones"}
+
+	page := 1
+
+	for {
+		url := strings.Join(strurl, ",")
+		client := &http.Client{}
+		re := regexp.MustCompile("^[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]$")
+		req, err := http.NewRequest("GET", url, nil)
+		if err != nil {
+			logger.Println(err)
+			break
+		}
+		req.Header.Add("PRIVATE-TOKEN", token)
+		resp, err := client.Do(req)
+		if err != nil {
+			logger.Println(err)
+			break
+		}
+		respByte, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			logger.Println("fail to read response data")
+			break
+		}
+		if project.State != "closed" && re.MatchString(project.Title) {
+
+			titleList := []string{"Title: ", project.Title, "due_date", project.DueDate}
+			y := strings.Join(titleList, ",")
+			list = append(list, y)
+		}
+
+		json.Unmarshal(respByte, project)
+		defer resp.Body.Close()
+
+		if project.Name == "" {
+			break
+		}
+		page++
```

Comment:
You could integrate the counter within the loop as shown in a previous comment by Antoine.

Reply:
My bad edited it in the other function but not in this one.